drivers/hud_path_report/spec/models/fy2024/path_context.rb

```diff
@@ -0,0 +1,81 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+RSpec.shared_context 'path context FY2024', shared_context: :metadata do
+  def shared_filter
+    {
+      start: Date.parse('2020-01-01'),
+      end: Date.parse('2020-12-31'),
```

Comment:
These should probably change eventually, but if the tests are passing now, let's leave them.